_posts/2019-02-22-Caring-For-The-Publisher.md

```diff
@@ -158,14 +163,15 @@ Now being able to use it like so:
 element.addEventListener('touchstart', function() {}, supportsPassive ? {passive: true} : false);
 ```
 
-The Passive Event listeners were mainly created for mobile and it is a game changer for 3rd party content as well, which is why we use them, to ensure that we do not impact the scrolling performance for the publishers that run our creatives.
+The Passive Event listeners were mainly created for mobile and it is a game changer for 3rd party content as well. This is why we use them, to ensure that we do not impact the scrolling performance for the publishers that run our creatives.
 
-Optimized assets and sprites
-Assets are usually the heaviest in size when delivering 3rd party content and need the extra attention. There are mainly two methods to approach image optimization, through spritesheets and compression. 
-SpriteSheet
+### Optimized assets and sprites 
+Assets are usually the heaviest in size when delivering 3rd party content and need the extra attention. There are mainly two methods to approach image optimization, through spritesheets and compression.  
+#### SpriteSheet
 A spritesheet is the combination of more images into a single larger one, this leads to less memory used and better performance by having less requests.
-Compression
-There are more modern image formats, such as WebP and JP2 (JPG 2000), that have a superior compression and better quality characteristics than their counterparts, JPG and PNG.
+#### Compression
+For lighter assets that don't require an alpha channel, we use JPG and we found that a compression of around 80% gives the best quality / file size ratio. As well, we strip all the metadata and use Progressive Encoding instead of the default "Baseline mode", for better UX.
```

Comment:
I would remove the `we found that a compression of...` - using values around ~80% is fairly standard and that's what most people use when it comes to optimizing images. So perhaps change it to `we use JPG with a standard compression rate of 80%` for optimal image quality / file size ratio`.

Also regarding the progressive encoding, instead of `for better UX` I would put `for faster display`. I find `for better UX` is very vague, we are not really sure that improves there.